CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Change Log
 
+## v1.4.0
+
+> Change log since v1.3.0
+
+### New Feature: JobSidecarTerminator
+
+In the Kubernetes world, it is challenging to use long-running sidecar containers for short-term application orchestration because there is no straightforward way to
+terminate the sidecar containers when the main container exits. For instance, when the main container in a Pod finishes its task and exits, it is expected that accompanying sidecars,
+such as a log collection sidecar, will also exit actively, so the Job Controller can accurately determine the completion status of the Pod.
+However, many sidecar containers may lack an active exit mechanism, and therefore, an additional solution is required to address this issue.
+
+For this scenario OpenKruise provides the JobSidecarTerminator capability, which can exit Sidecar containers after the Main container exits.
+
+For more detail, please refer to its [proposal](https://github.com/openkruise/kruise/blob/master/docs/proposals/20230130-job-sidecar-terminator.md).
+
+### ContainerRecreateRequest
+- Add 'forceRecreate' field to ensure that some extreme scenarios can also achieve the restart of the container. ([#1182](https://github.com/openkruise/kruise/pull/1182), [@BH4AWS](https://github.com/BH4AWS))
```

Comment:
> Add 'forceRecreate' field to ensure that some extreme scenarios can also achieve the restart of the container. 
 
Add 'forceRecreate' field to ensure the immediate recreation of the container even if the container is starting at that point